src/main/java/com/github/harti2006/neo4j/StartNeo4jServerMojo.java

```diff
@@ -102,6 +122,17 @@ private void startNeo4jServer() throws MojoExecutionException {
         final Log log = getLog();
         try {
             final Path serverLocation = getServerLocation();
+            
+            // Delete existing DB if required
+            if ( deleteDb )
+            {
+            		Path dbdir = serverLocation.resolve ( "data/databases/graph.db" );
+            		log.info ( "Deleting Database directory: '" + dbdir.toAbsolutePath ().toString () + "'" );
+	            FileUtils.deleteQuietly ( 
+	            		serverLocation.resolve ( "data" ).toFile ()
```

Comment:
This is a bit confusing for users: The log message says it deletes `data/databases/graph.db`, but actually the entire `data` directory is removed.

Reply:
Will fix the message. I need to delete the whole databases directory, because otherwise former authentication tokens are kept in auth/ and the newly created graph.db is not accessible with the default neo4j/neo4j